tests/test_content_section.py

```diff
@@ -190,3 +192,23 @@ def test_filtering_a_section_with_a_description_template(self):
 
         assert section.filter({"lot": "digital-specialists"}).description == "description for specialists"
         assert section.filter({"lot": "digital-outcomes"}).description == "description for outcomes"
+
+    def test_question_followup_get_data(self):
+        section = ContentSection(
+            slug='section',
+            name=TemplateField('Section one'),
+            prefill=False,
+            editable=False,
+            edit_questions=False,
+            questions=[Question({'id': 'q1', 'followup': {'q2': [True]}, 'type': 'boolean'}),
```

Comment:
Good call, I've fliped the values here. Followups can be attached to radios and checkboxes now too.